src/modules/75-cd/components/PipelineSteps/Common/Terraform/Editview/TerraformVarfile.module.scss

```
@@ -296,3 +296,7 @@
 .newConnectorButton {
   margin-top: 40px !important;
 }
+
+.tooltipProps {
```

Not a great name. Maybe something more descriptive
```suggestion
.repoNameButtonTooltip {
```